python/sglang/srt/server_args.py

```
@@ -383,6 +383,18 @@ def __post_init__(self):
             self.disable_overlap_schedule = True
             logger.warning("Overlap scheduler is disabled for decode server")
 
+        if self.enable_memory_saver:
+            try:
+                import torch_memory_saver
+            except ImportError:
+                logger.warning(
+                    "enable_memory_saver is enabled, but "
+                    "torch-memory-saver is not installed. Please install it "
+                    "via `pip3 uninstall torch-memory-saver`. "
+                    "For normal operation, it will be disabled."
+                )
+                self.enable_memory_saver = False
```

it should raise an error.